eclair-node/src/main/scala/fr/acinq/eclair/api/handlers/PathFinding.scala

```diff
@@ -56,12 +56,18 @@ trait PathFinding {
     }
   }
 
+  val node: Route = postRequest("node") { implicit t =>
```

Comment:
I called this `node` for consistency with the `nodes` API, but it's a bit weird...should I rename this?

Reply:
It looks like LND uses [`GetNodeInfo`](https://lightning.engineering/api-docs/api/lnd/lightning/get-node-info) or just `node` for REST and I couldn't find an equivalent in [CLN](https://lightning.readthedocs.io/lightning-help.7.html).

Using `node` seems fine for our API.

Reply:
Great, if `lnd` calls this API `node` we should do the same, consistency will help cross-implementation devs. Thanks for checking that!